plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/webview/Browser.kt

```diff
@@ -92,7 +99,8 @@ class Browser(parent: Disposable) : Disposable {
                         $isCodeTransformAvailable, // whether /transform is available
                         $isDocAvailable, // whether /doc is available
                         $isCodeScanAvailable, // whether /scan is available
-                        $isCodeTestAvailable // whether /test is available
+                        $isCodeTestAvailable, // whether /test is available
+                        ${jacksonObjectMapper().writeValueAsString(highlightCommand)}
```

Comment:
can you bump the `jacksonObjectMapper()` to the companion object as a private val so it can be reused?